app/controllers/concerns/company_favourites_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CompanyFavouritesController < ApplicationController
```

Comment:
why are these in the `concerns` directory? Never been sure what that dir was for tbh...